db/migrate/20170707205938_default_value_for_reduction_subgroup.rb

```diff
@@ -0,0 +1,6 @@
+class DefaultValueForReductionSubgroup < ActiveRecord::Migration[5.1]
+  def change
+    change_column :reductions, :subgroup, :string, :default => "_default"
+    Reduction.update_all(:subgroup => "_default")
```

Comment:
Could you add a `change_column_null :reductions, :subgroup, false` after this line?

Reply:
Resolved this, but of course github doesn't let me mark that :)